app/templates/selectbox/index.js

```diff
@@ -6,54 +6,28 @@ module.exports = function(selected, options, translate, changed) {
   let x = selected;
 
   return html`
-  <div class="selectbox">
-    <div onclick=${toggle}>
-      <span class="link">${translate(selected)}</span>
-      <svg width="32" height="32">
-        <polygon points="8 18 17 28 26 18" fill="#0094fb"/>
-      </svg>
-    </div>
-    <ul id="${id}" class="selectbox__options">
+  <span class="select">
+    <select id="${id}">
+      <option>${translate(selected)}</option>
       ${options.map(
-        i => html`
-            <li
-              class="selectbox__option"
-              onclick=${choose}
-              data-value="${i}">${number(i)}</li>`
+        i =>
+          html`<option value="${i}" onclick=${choose}>${translate(
+            number(i)
+          )}</option>`
       )}
-    </ul>
-  </div>`;
-
-  function close() {
-    const ul = document.getElementById(id);
-    const body = document.querySelector('body');
-    ul.classList.remove('selectbox__options--active');
-    body.removeEventListener('click', close);
-  }
-
-  function toggle(event) {
-    event.stopPropagation();
-    const ul = document.getElementById(id);
-    if (ul.classList.contains('selectbox__options--active')) {
-      close();
-    } else {
-      ul.classList.add('selectbox__options--active');
-      const body = document.querySelector('body');
-      body.addEventListener('click', close);
-    }
-  }
+    </select>
+    <svg id="arrow" width="32" height="32">
+      <polygon points="8 18 17 28 26 18" fill="#0094fb"/>
+    </svg>
+  </span>`;
```

Comment:
I'd suggest using `onchange` for the event handler, and `selected` on the selected option. Something like:

```js
html`
  <span class="select">
    <select id="${id}" onchange=${choose}>
      ${options.map(
        i =>
          html`<option value="${i}" ${i === selected ? 'selected' : ''}>${translate(i)}</option>`
      )}
    </select>
  </span>`;
```

Reply:
Of course, that is much more clearer.